packages/transformer/src/test/standalone/IModelTransformerHub.test.ts

```diff
@@ -1662,6 +1662,122 @@ describe("IModelTransformerHub", () => {
     sinon.restore();
   });
 
+  it("should be able to handle a transformation which deletes a relationship and then elements of that relationship", async () => {
+    const masterIModelName = "Master";
+    const masterSeedFileName = path.join(outputDir, `${masterIModelName}.bim`);
+    if (IModelJsFs.existsSync(masterSeedFileName))
+      IModelJsFs.removeSync(masterSeedFileName);
+    // userlabel / codevalue: set json property named updateState on the physicalobject to value (simplified state updates)
+    const masterSeedState = {40:1, 2:2, 41:3, 42:4};
+    const masterSeedDb = SnapshotDb.createEmpty(masterSeedFileName, { rootSubject: { name: masterIModelName } });
+    masterSeedDb.nativeDb.setITwinId(iTwinId); // workaround for "ContextId was not properly setup in the checkpoint" issue
+    populateTimelineSeed(masterSeedDb, masterSeedState);
+
+    const noFedGuidElemIds = masterSeedDb.queryEntityIds({ from: "Bis.Element", where: "UserLabel IN (41, 42)" });
```

Comment:
I guess this works? user label is a string, so you should make 41/42 string literals anyway `('41', '42')`, and if it's a copy of code I wrote, fix that too haha

Reply:
changed it in both places